test/services/nexboard/index.test.js

```diff
@@ -209,4 +209,135 @@ describe('Nexboard services', () => {
 		expect(board).to.be.an('object');
 		expect(board.id).to.equal(id);
 	});
+
+	it('Find should need TOOL_VIEW permission.', async () => {
+		const name = `${Date.now()}testRole`;
+		const role = await testHelpers.createTestRole({ name, permissions: [] });
+
+		const {
+			requestParams: { authentication: { accessToken } },
+		} = await testHelpers.setupUser({ roles: [name] });
+
+		const data = {
+			title: '1',
+			description: '2',
+			projectId: '1234',
+		};
+
+		const { body } = await request({
+			server: app,
+			endpoint: '/nexboard/boards',
+			method: 'get',
+			data,
+			accessToken,
+		});
+
+		expect(body.code).to.equal(403);
+	});
+
+	it('Get should need TOOL_VIEW permission.', async () => {
+		const name = `${Date.now()}testRole`;
+		const role = await testHelpers.createTestRole({ name, permissions: [] });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['role' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5394667)